### PR TITLE
I got vimeo up to spec

### PR DESCRIPTION
**Pull request by rafirosenberg**
Pretty much the way i remember your wanting it ... there are some thing we can do to make it more elegant such as hiding the video parts of the form when we don't need them as well as distinguishing video pins from image pins.. but i want to know what you think so far ...

Also i couldn't get youtube to work because the bookmarkelt wont work at all on the youtube site and I'm not sure why .. once we get that working i can try to make youtube work ... 

ps on vimeo it picks up images as  images and videos as videos ...

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pinry.settings.development")
+    sys.path.insert(0, './lib/python2.7/site-packages')
+
+    print(sys.path)
```
Comment (Delphinator) [inline manage.py]: This basically enables the virtualenv. I see two issues with it:

1. It has nothing to do with the vimeo integration.
2. It breaks if I choose to install the virtualenv anywhere else than in the root folder.

File: pinry/core/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,98 +0,0 @@
...
-        u'taggit.taggeditem': {
-            'Meta': {'object_name': 'TaggedItem'},
-            'content_type': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "u'taggit_taggeditem_tagged_items'", 'to': u"orm['contenttypes.ContentType']"}),
-            u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'object_id': ('django.db.models.fields.IntegerField', [], {'db_index': 'True'}),
-            'tag': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "u'taggit_taggeditem_items'", 'to': u"orm['taggit.Tag']"})
-        }
-    }
-
-    complete_apps = ['core']
```
Comment (Delphinator) [inline pinry/core/migrations/0001_initial.py]: Isn't this needed to initialize the database?

File: pinry/core/models.py [hunk 3]
```diff
@@ -41,6 +41,8 @@ class Pin(models.Model):
     url = models.URLField(null=True)
     origin = models.URLField(null=True)
     description = models.TextField(blank=True, null=True)
+    video_id = models.CharField(max_length=30, blank=True, null=True)
+    service = models.CharField(max_length=35, blank=True, null=True)
```
Comment (Delphinator) [inline pinry/core/models.py]: This needs a migration to support existing instalations!

File: pinry/settings/__init__.py [hunk 4]
```diff
@@ -7,9 +7,9 @@
 
 
 # Set to False to disable people from creating new accounts.
-ALLOW_NEW_REGISTRATIONS = False
+ALLOW_NEW_REGISTRATIONS = True
 
-# Set to False to force users to login before seeing any pins. 
+# Set to False to force users to login before seeing any pins.
```
Comment (Delphinator) [inline pinry/settings/__init__.py]: Like in `manage.py`: This is unrelated to the vimeo integration.

File: pinry/settings/development.py [hunk 5]
```diff
@@ -13,4 +13,11 @@
     }
 }
 
+INSTALLED_APPS += (
+)
+
+MIDDLEWARE_CLASSES += (
+     'django.contrib.auth.middleware.AuthenticationMiddleware',
+)
+
```
Comment (Delphinator) [inline pinry/settings/development.py]: Like in manage.py: This is unrelated to the vimeo integration.

File: pinry/static/js/bookmarklet.js [hunk 6]
```diff
@@ -1,15 +1,13 @@
...
-
+* Bookmarklet for Pinry
+* Descrip: This is trying to be as standalone a script as possible hence
+* why it has built in helpers and such when the rest of the
+* scripts make use of helpers.js. In the future i want to remove
+* all dependencies on jQuery.
+* Authors: Pinry Contributors
+* Updated: Mar 4th, 2013
+* Require: None (dynamically loads jQuery if needed)
+*/
```
Comment (Delphinator) [inline pinry/static/js/bookmarklet.js]: The formatting looked better before. And like in manage.py: This is unrelated to the vimeo integration.

File: pinry/templates/includes/pins.html~ [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+                    {{#each tags}}
+                        <span class="tag"><a href="/pins/tag/{{this}}/">{{this}}</a></span>
+                    {{/each}}
+                    {{/if}}
+                </div>
+            </div>
+        </div>
+        {{/each}}
+    </script>
+{% endverbatim %}
```
Comment (Delphinator) [inline pinry/templates/includes/pins.html~]: This is a temporary file from your editor.

**Comment by Delphinator** [comment]
Awesome idea! I went through the diff and commented it. I tried to make it as constructive as possible, I'm sorry if it sounds negative (I'm German, we're too blunt sometimes).

**Comment by overshard** [comment]
I'm going to go through and test this while I'm at it as well, overall seems like good work but needs cleaned up. I'll see what I can do to make it merge though.
